common/include/villas/kernel/devices/utils.hpp

```diff
@@ -0,0 +1,26 @@
+/* Utils
+ *
+ * Author: Pascal Bauer <pascal.bauer@rwth-aachen.de>
+ *
+ * SPDX-FileCopyrightText: 2023-2024 Pascal Bauer <pascal.bauer@rwth-aachen.de>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+#include <vector>
+
+namespace villas {
+namespace kernel {
+namespace devices {
+namespace utils {
+
+void write_to_file(std::string data, const std::filesystem::path file);
```

Comment:
doesn't it make sense to have this somewhere more general? @stv0g maybe you know a good place? `common/include/villas/utils` maybe?

Reply:
I agree. We already have https://github.com/VILLASframework/node/blob/master/common/lib/utils.cpp